1-JavaScript/Week2/homeworkSolition/ex6-sol-totalCost.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const cartForParty = {
+  beers: 1.75,
+  chips: 0.99,
+  chikenwings: 2.433,
+  water: 2,
+  bread: 1.5,
+};
+
+function calculateTotalPrice(price) {
```

Comment:
I would not name this parameter `price`, because you want to be able to put in a complete object. So the naming is a bit confusing 🙂 You could make it `cart` for example.

```suggestion
function calculateTotalPrice(cart) {
```